CHANGES.rst

```diff
@@ -367,6 +369,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
```

Comment:
That's why it's better to wait for a maintainer to milestone the PR before adding the changelog entry. Also please change the PR number, it should point to the PR  (#4782) and not the issue it fixes.